src/features/wallet/root/ActivityDetails/Rewards.tsx

```diff
@@ -0,0 +1,32 @@
+import { AnyTransaction, PendingTransaction, RewardsV1 } from '@helium/http'
+import React from 'react'
+import { groupBy } from 'lodash'
+import ActivityRewardItem from './ActivityRewardItem'
+import Box from '../../../../components/Box'
+
+type Props = { item: AnyTransaction | PendingTransaction }
+const Rewards = ({ item }: Props) => {
+  if (item.type !== 'rewards_v1') return null
+
+  const rewards = item as RewardsV1
+
+  const grouped = groupBy(rewards.rewards, (reward) => reward.type)
+
+  return (
+    <Box marginTop="lx">
+      {Object.keys(grouped).map((k, idx) =>
```

Comment:
I think this may differ from the design, not sure if intentional. In the design, there's a single pod per hotspot, and its various rewards are listed below it.

![image](https://user-images.githubusercontent.com/792845/106041744-0bd58880-6091-11eb-9477-297f7c7860ae.png)

compared to a row per reward, grouped by reward type

![image](https://user-images.githubusercontent.com/792845/106041785-1d1e9500-6091-11eb-9a7a-629689c01d88.png)

Reply:
I'm following you. No, this was not an intentional switch. I'll fix it up. Good catch ✔️